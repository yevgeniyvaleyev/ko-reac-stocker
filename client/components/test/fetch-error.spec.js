import React from 'react';
import renderer from 'react-test-renderer';
import FetchError from '../fetch-error';


test('should render component', () => {
  const data = {
    message: 'test',
    onRetry: () => {},
  };
  const component = renderer.create(
    <FetchError {...data} />,
  );
  const tree = component.toJSON();

  expect(tree).toMatchSnapshot();
});
