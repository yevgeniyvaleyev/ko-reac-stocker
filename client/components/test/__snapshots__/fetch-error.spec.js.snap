// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render component 1`] = `
<div
  className="sc-bdVaJa jZnjlv"
  style={
    Object {
      "WebkitTapHighlightColor": "rgba(0,0,0,0)",
      "backgroundColor": "#ffffff",
      "borderRadius": 2,
      "boxShadow": "0 1px 6px rgba(0, 0, 0, 0.12),
         0 1px 4px rgba(0, 0, 0, 0.12)",
      "boxSizing": "border-box",
      "color": "rgba(0, 0, 0, 0.87)",
      "fontFamily": "Roboto, sans-serif",
      "muiPrepared": true,
      "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
    }
  }
>
  <span>
    Can't fetch stocks. 
    test
  </span>
  <button
    disabled={false}
    href={undefined}
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    onTouchStart={[Function]}
    style={
      Object {
        "WebkitTapHighlightColor": "rgba(0, 0, 0, 0)",
        "backgroundColor": "rgba(0, 0, 0, 0)",
        "border": 10,
        "borderRadius": 2,
        "boxSizing": "border-box",
        "color": "#00bcd4",
        "cursor": "pointer",
        "display": "inline-block",
        "fontFamily": "Roboto, sans-serif",
        "fontSize": "inherit",
        "fontWeight": "inherit",
        "height": 36,
        "lineHeight": "36px",
        "margin": 0,
        "minWidth": 88,
        "muiPrepared": true,
        "outline": "none",
        "overflow": "hidden",
        "padding": 0,
        "position": "relative",
        "textAlign": "center",
        "textDecoration": "none",
        "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
        "userSelect": "none",
        "verticalAlign": null,
        "zIndex": 1,
      }
    }
    tabIndex={0}
    type="button"
  >
    <div
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchStart={[Function]}
    >
      <span
        style={
          Object {
            "fontSize": 14,
            "fontWeight": 500,
            "letterSpacing": 0,
            "muiPrepared": true,
            "paddingLeft": 16,
            "paddingRight": 16,
            "position": "relative",
            "textTransform": "uppercase",
            "verticalAlign": "middle",
          }
        }
      >
        Retry
      </span>
    </div>
  </button>
</div>
`;
